client/myexploratory.Rmd

```diff
@@ -0,0 +1,4 @@
+```{python}
+
+```
+
```

Comment:
I'm not sure what this file does - can we remove it?

Reply:
Done in https://github.com/wrattler/wrattler/pull/69